### PR TITLE
Name and UID glossary terms

### PR DESCRIPTION
**Pull request by tonglil**
Extract glossary terms from page.

`Name` is part of https://github.com/kubernetes/website/issues/5993.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6606)

### CODE REVIEW

File: _data/glossary/name.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+id: name
+name: Name
+full-link: /docs/concepts/overview/working-with-objects/names
+tags:
+  - fundamental
+short-description: >
+  A client-provided string used to refer to an object in a resource URL, such as `/api/v1/pods/some-name`.
```
Comment (zacharysarah) [inline _data/glossary/name.yaml]: ```A client-provided string that refers to an object in a resource URL, such as `/api/v1/pods/some-name`.
```

File: _data/glossary/name.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+long-description: >
+  Only one object of a given kind can have a given name at a time (i.e., they are spatially unique).
```
Comment (zacharysarah) [inline _data/glossary/name.yaml]:
```
Only one object of a given kind can have a given name at a time.
```

File: _data/glossary/name.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  But if you delete the object, you can make a new object with the same name.
```
Comment (zacharysarah) [inline _data/glossary/name.yaml]:
```
However, if you delete the object, you can make a new object with the same name.
```

File: _data/glossary/uid.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+id: uid
+name: UID
+full-link: /docs/concepts/overview/working-with-objects/names
+tags:
+  - fundamental
+short-description: >
+  A Kubernetes systems-generated string to uniquely identify objects.
+long-description: >
+  Every object created over the whole lifetime of a Kubernetes cluster has a distinct UID (i.e., they are spatially and temporally unique).
```
Comment (zacharysarah) [inline _data/glossary/uid.yaml]:
```
Every object created over the whole lifetime of a Kubernetes cluster has a distinct UID.
```

**Comment by tonglil** [comment]
Updated!

**Pull request merged**